### ISSUE TITLE
Execute stream group actions from left menu

### ISSUE THREAD

**Issue by sliechti** [sliechti/feedrdr#54]
For example mark all as read

**Comment by Nidhi0257** [comment]
mark all read is working in production.
when you select stream and click on mark all read, message appear "Finished reading stream" and the Stream name disappears.

**Comment by sliechti** [comment]
Yes, I'm thinking about a way to execute those actions directly from the left bar, without having to enter the stream. 
I'm working on a new interim design. let's put this one on hold for now.
